docs/rules/no-alert.md

```diff
@@ -1,6 +1,6 @@
 # Disallow Use of Alert (no-alert)
 
-JavaScripts' alert, confirm, and prompt functions are widely considered to be obtrusive as UI elements and should be replaced by a more appropriate custom UI implementation. Furthermore, alert is often used while debugging code, which should be removed before deployment to production.
+JavaScript's `alert`, `confirm`, and `prompt` functions are widely considered to be obtrusive as UI elements and should be replaced by a more appropriate custom UI implementation. Furthermore, `alert` is often used while debugging code, which should be removed before deployment to production.
```

Comment:
These fixes should be removed from here, since we've already merged your other PR that fixes these